.travis.yml

```diff
@@ -1,35 +1,52 @@
+services:
+  - docker
 
 dist: trusty
 language: python
-python:
-  - "3.6"
-
-addons:
-  apt:
-    sources:
-      - deadsnakes
-      - ubuntu-toolchain-r-test
-    packages:
-      - gcc-8
-      - g++-8
-      - cmake
 
 install:
-  - pip install --upgrade setuptools pytest
+  - docker run -e PYABI=$PYABI -v $(pwd):/pytomlpp -w /pytomlpp quay.io/pypa/manylinux2010_x86_64 ci/build-wheel.sh wheels/
+  - ls wheels/
+  - pip install -U pip setuptools wheel
+  - pip install wheels/*.whl
+  - pip install pytest
+
+script: pytest -vvra
 
-script:
-  # c++ part, build and run c++ unit tests
-  - export CC=gcc-8
-  - export CXX=g++-8
-  - mkdir -p build
-  - cd build
-  - cmake ..
-  - make -j4
-  - make test
-  - cd ..
+jobs:
+  include:
+    - python: 2.7
```

Comment:
I had a look at the most recent CI build on this branch and it seems the job is failing for Python 2.7 as I used some of the Python 3 features in my cpp code. (https://github.com/bobfang1992/pytomlpp/blob/master/src/pytomlpp.hpp#L39, timezone class in datetime module)

My suggestion is that we remove Python 2.7 support altogether. As it is not officially supported anyway and I would imagine people will migrate to Python 3. 

What's your thought?

Reply:
If this is to be used in packaging, you'll likely need to support Python 2.7 until January 2021. If you don't have Python 2, some packaging libraries can't use your module. This decision is up to you.

Reply:
Hi, what do you mean by packaging projects? I am working on this branch now to add py2 support.